modules/openy_features/openy_media/openy_media.libraries.yml

```diff
@@ -9,3 +9,8 @@ browser:
     - core/jquery
     - core/drupal
     - core/underscore
+
+blazy_ratio:
+  css:
```

Comment:
let's add version for proper cache invalidation

Reply:
fixed, thanks